build.sbt

```
@@ -22,7 +24,15 @@ lazy val rootProject = (project in file(".")).settings(
       "ch.qos.logback" % "logback-classic" % Versions.logback,
       "com.softwaremill.sttp.tapir" %% "tapir-sttp-stub-server" % Versions.tapir % Test,
       "org.scalatest" %% "scalatest" % Versions.scalaTest % Test,
-      "com.softwaremill.sttp.client3" %% "circe" % Versions.circe % Test
+      "com.softwaremill.sttp.client3" %% "circe" % Versions.circe % Test,
+      "io.circe" %% "circe-refined" % "0.14.5",
+      "eu.timepit" %% "refined" % Versions.refined,
+      "org.tpolecat" %% "doobie-core" % Versions.doobie,
+      "org.tpolecat" %% "doobie-refined" % Versions.doobie,
+      "org.tpolecat" %% "doobie-postgres" % Versions.doobie,
+      "org.tpolecat" %% "doobie-scalatest" % Versions.doobie % "test",
```

To make things uniform, I think this is the preferred approach
```suggestion
      "org.tpolecat" %% "doobie-scalatest" % Versions.doobie % Test,
```